src/wallet/api/wallet2_api.h

```diff
@@ -445,6 +445,12 @@ struct Wallet
         ConnectionStatus_WrongVersion
     };
 
+    enum BackgroundSyncType {
+        BackgroundSync_Off = 0,
+        BackgroundSync_ReusePassword = 1,
+        BackgroundSync_CustomPassword = 2
+    };
+
```

Comment:
> Just wondering: Is this needed in addition to the enum that exists already in wallet2 because the "API" avoids mixing own things with wallet2 things?

Right, the GUI doesn't include wallet2.h directly, it's exposed to wallet2 via this wallet2_api.h. I'm open to a saner approach instead of duplicating this enum logic if you've got one in mind

> And do the names have underscores here, but none in wallet2, because again we are in another environment, with other naming conventions?

Right, see wallet2_api.h 's `ConnectionStatus` versus wallet2.h's `AskPasswordType`.